Makefile

```diff
@@ -31,11 +31,6 @@ ifneq ($(wildcard build/custom.mk),)
 	include build/custom.mk
 endif
 
-## Propagates plugin manifest information into the server/ and webapp/ folders as required.
-.PHONY: apply
-apply:
-	./build/bin/manifest apply
-
```

Comment:
This target works correctly, doesn't it? If so, we should remove https://github.com/mattermost/mattermost-plugin-azure-devops/blob/788729d03e793f44b2211b25d1328ace0581e299/build/legacy.mk#L1-L4 not this one.

Reply:
@hanzei Okay that sounds correct given the current circumstances. Why do we need this though? I thought we created `legacy.mk` to avoid needing to run `make apply`